### DISCUSSION TITLE
What would be some good additional filters?

### DISCUSSION THREAD

**Discussion by raae** [queen-raae/prune-your-follows#10] [Ideas]

**Comment by ccoenen** [comment]
"Account unused since". Not the tweets per year metric, but "hasn't tweeted in X years". this should be fairly easy to obtain, just get the most recent tweets of a person and sort by date.

**Reply by ccoenen** [reply]
in a similar vein, the _overall_ tweet activity in tweets per year may be deceptive. The account may have been dormant for a while but may now spam your timeline.

If possible, the **recent activity** would be interesting as well. Maybe a proxy would be to get the most recent 100 Tweets, and see over which timeframe they occurred. 100 tweets over three months -> pretty low recent frequency. 100 tweets over the past two days -> holy crap, two per hour!

**Comment by ccoenen** [comment]
"original content": some people just do 100% retweets of other sources. Some post only original content.

**Comment by ccoenen** [comment]
accounts that _never_ interact with others vs. accounts that _mostly_ just @-mention other accounts (and rarely post regular tweets)

**Comment by ccoenen** [comment]
Accounts that just dump one external link after another (like newsbots)

**Comment by ccoenen** [comment]
one might even use twitter's ♥/⭐ feature to see, which accounts I _often_ fav/star and which accounts I never fav/star. I'm not sure if this is easy to to do, and I'm not even sure what it means, so take it with a grain of salt :-D

(slightly inspired by #7)

**Comment by JudoHacker** [comment]
I would like to see in each card whether that person follows me.  Or perhaps this could be a global filter to hide all people who follow me. 🤷

**Comment by CasJam** [comment]
Inactivity of a user is less important to me (if they're inactive, I don't see their content much, so that's not a "pain").

Being able to unfollow people who I _do often_ see in my feed but don't want to see anymore is more useful / solves a pain.  I'm sure it's harder to build this, of course :)

I'm most interested in:
- People I never like or engage with their tweets
- People I haven't added to private lists
- People I followed a long time ago and maybe lost interest in lately.

**Reply by raae** [reply]
Thank you for the feedback. The idea came from me hitting the 5000 limit and wanting to find folks I could unfollow without much impact on my feed. Interesting that your "pain" is the complete opposite. I'll have to do some exploration of the Twitter API to see what is feasible 🤔

**Reply by JudoHacker** [reply]
I really like the "People I never like or engage with their tweets" idea.  I would also like "People who have never liked or engage with my tweets".

**Comment by whitep4nth3r** [comment]
I'd like to see if I have "muted" someone on the cards, just for reference. Just because sometimes I like to keep following people but not see tweets regularly. For reasons 😄

**Reply by raae** [reply]
I do this too 😬

**Reply by mjwhansen** [reply]
yes, same! Especially people who are personal friends/former coworkers who tweet a lot or on things that don't interest me. I still want to follow them, but not see them in my TL.

**Comment by raae** [comment]
Empty descriptions could be a good indicator someone is not very serious about their Twitter presence!

**Comment by mjwhansen** [comment]
- showing whether someone follows me back
- showing whether I've muted someone
- ability to mute someone from the app
- ability to check whether i've turned off retweets and if not, turn off retweets button (I've turned off retweets for the vast majority of my follows. It's my default behavior now when I follow someone new, or see a lazy RT in my feed. Cuts down on the noise SIGNIFICANTLY)
